### PR TITLE
Make source release for ome-common when building from git

### PR DESCRIPTION
**Pull request by ghost**
Use the script added in https://github.com/ome/ome-common-cpp/pull/12 to create a source release for ome-common-cpp.

---

Testing: Check the sources in https://ci.openmicroscopy.org/view/Files-DEV/job/OME-FILES-CPP-DEV-merge-superbuild/BUILD_TYPE=Release,node=cowfish/lastSuccessfulBuild/artifact/artefacts/sources/
This should include tar.xz and zip files for ome-common-cpp.  These should download and build, and contain embedded version information in cmake/GitVersion.cmake.

### CODE REVIEW

**Comment by ghost** [comment]
Example: https://ci.openmicroscopy.org/job/OME-FILES-CPP-DEV-merge/BUILD_TYPE=Release,node=cod/lastSuccessfulBuild/artifact/artefacts/sources/
https://ci.openmicroscopy.org/job/OME-FILES-CPP-DEV-merge-superbuild/BUILD_TYPE=Release,node=cowfish/lastSuccessfulBuild/artifact/artefacts/sources/

File: scripts/jenkins-build [hunk 1]
```diff
@@ -178,8 +179,22 @@ build() {
     done
 
     # Source release
-    echo "Making source release"
+    echo "Making source release for ome-cmake-superbuild"
     make_source_release "$OME_VERSION" "$git_branch"
+    echo "Making source release for ome-cmake-superbuild: done"
+
+    if [ "$build_git" = "ON" ];then
+        for component in ome-common; do
```
Comment (joshmoore) [inline scripts/jenkins-build:187]: This is the line you eventually assume will contain all of our own components?

**Comment by joshmoore** [comment]
Check of the linked zips is good. See https://github.com/ome/ome-common-cpp/pull/12#issuecomment-197803628 for discussion.

File: scripts/jenkins-build (hunk 1 above)
Comment (ghost) [inline scripts/jenkins-build:187]: Yes

**Comment by ghost** [comment]
@joshmoore I've opened https://github.com/ome/ome-cmake-superbuild/pull/45 and https://github.com/ome/ome-files-cpp/pull/3 to follow this up for the ome-files and ome-cmake-superbuild source releases.  The PRs you reviewed here (this one and https://github.com/ome/ome-common-cpp/pull/12) are mergeable before these two, but equally all four can be merged as a set if you want to delay merging.

**Comment by joshmoore** [comment]
This was automated merged when #45 was merged based on https://github.com/ome/ome-common-cpp/pull/12#issuecomment-198321680

In the future, please close redundant PRs to clarify the situation.

**Pull request merged**